diff --git a/gitk-git/gitk.orig b/gitk-git/gitk
index ce63a8c..4ae5a56 100755
--- a/gitk-git/gitk.orig
+++ b/gitk-git/gitk
@@ -9150,7 +9150,7 @@ proc cherrypick {} {
     update
     # Unfortunately git-cherry-pick writes stuff to stderr even when
     # no error occurs, and exec takes that as an indication of error...
-    if {[catch {exec sh -c "git cherry-pick -r $rowmenuid 2>&1"} err]} {
+    if {[catch {exec sh -c "git cherry-pick -x -r $rowmenuid 2>&1"} err]} {
 	notbusy cherrypick
 	if {[regexp -line \
 		 {Entry '(.*)' (would be overwritten by merge|not uptodate)} \
