diff -urN git-1.5.0.org/Makefile git-1.5.0/Makefile
--- git-1.5.0.org/Makefile	2007-02-14 03:46:46.000000000 +0100
+++ git-1.5.0/Makefile	2007-02-14 10:15:35.646623000 +0100
@@ -614,7 +614,7 @@
 
 all::
 	$(MAKE) -C git-gui all
-	$(MAKE) -C perl PERL_PATH='$(PERL_PATH_SQ)' prefix='$(prefix_SQ)' all
+	$(MAKE) -C perl PERL_PATH='$(PERL_PATH_SQ)' prefix='$(prefix_SQ)' INSTALLDIRS='$(INSTALLDIRS)' all
 	$(MAKE) -C templates
 
 strip: $(PROGRAMS) git$X
@@ -647,7 +647,7 @@
 $(patsubst %.perl,%,$(SCRIPT_PERL)): perl/perl.mak
 
 perl/perl.mak: GIT-CFLAGS
-	$(MAKE) -C perl PERL_PATH='$(PERL_PATH_SQ)' prefix='$(prefix_SQ)' $(@F)
+	$(MAKE) -C perl PERL_PATH='$(PERL_PATH_SQ)' prefix='$(prefix_SQ)' INSTALLDIRS='$(INSTALLDIRS)' $(@F)
 
 $(patsubst %.perl,%,$(SCRIPT_PERL)): % : %.perl
 	rm -f $@ $@+
@@ -780,6 +780,7 @@
 
 perl/Makefile: perl/Git.pm perl/Makefile.PL GIT-CFLAGS
 	(cd perl && $(PERL_PATH) Makefile.PL \
+		INSTALLDIRS='$(INSTALLDIRS)' \
 		PREFIX='$(prefix_SQ)')
 
 doc:
