diff -ur git-2.12.1.orig/t/t5615-alternate-env.sh git-2.12.1/t/t5615-alternate-env.sh
--- git-2.12.1.orig/t/t5615-alternate-env.sh	2017-03-22 18:52:45.344354935 +0100
+++ git-2.12.1/t/t5615-alternate-env.sh	2017-03-22 18:52:59.320917887 +0100
@@ -77,6 +77,7 @@
 	check_obj "$quoted:$unquoted" <<-EOF
 	$one blob
 	$two blob
+	EOF
 '
 
 test_expect_success !MINGW 'broken quoting falls back to interpreting raw' '
diff -ur git-2.12.1.orig/t/t7004-tag.sh git-2.12.1/t/t7004-tag.sh
--- git-2.12.1.orig/t/t7004-tag.sh	2017-03-22 18:52:45.347688243 +0100
+++ git-2.12.1/t/t7004-tag.sh	2017-03-22 18:53:28.757366131 +0100
@@ -880,21 +880,13 @@
 '
 
 test_expect_success 'verifying a proper tag with --format pass and format accordingly' '
-	cat >expect <<-\EOF
+	cat >expect <<-\EOF &&
 	tagname : signed-tag
-	EOF &&
+	EOF
 	git tag -v --format="tagname : %(tag)" "signed-tag" >actual &&
 	test_cmp expect actual
 '
 
-test_expect_success 'verifying a forged tag with --format fail and format accordingly' '
-	cat >expect <<-\EOF
-	tagname : forged-tag
-	EOF &&
-	test_must_fail git tag -v --format="tagname : %(tag)" "forged-tag" >actual &&
-	test_cmp expect actual
-'
-
 # blank and empty messages for signed tags:
 
 get_tag_header empty-signed-tag $commit commit $time >expect
--- git-2.12.1.orig/t/t7030-verify-tag.sh	2017-03-22 18:52:45.347688243 +0100
+++ git-2.12.1/t/t7030-verify-tag.sh	2017-03-22 19:06:41.894375653 +0100
@@ -126,19 +126,11 @@
 '
 
 test_expect_success 'verifying tag with --format' '
-	cat >expect <<-\EOF
+	cat >expect <<-\EOF &&
 	tagname : fourth-signed
-	EOF &&
+	EOF
 	git verify-tag --format="tagname : %(tag)" "fourth-signed" >actual &&
 	test_cmp expect actual
 '
 
-test_expect_success 'verifying a forged tag with --format fail and format accordingly' '
-	cat >expect <<-\EOF
-	tagname : 7th forged-signed
-	EOF &&
-	test_must_fail git verify-tag --format="tagname : %(tag)" $(cat forged1.tag) >actual-forged &&
-	test_cmp expect actual-forged
-'
-
 test_done
--- git-2.12.1.orig/t/t7406-submodule-update.sh	2017-03-22 18:52:45.347688243 +0100
+++ git-2.12.1/t/t7406-submodule-update.sh	2017-03-22 19:16:48.043071963 +0100
@@ -440,17 +440,6 @@
 	) &&
 	test_i18ncmp actual expect
 '
-
-test_expect_success 'submodule update - command run for initial population of submodule' '
-	cat <<-\ EOF >expect
-	Execution of '\''false $submodulesha1'\'' failed in submodule path '\''submodule'\''
-	EOF &&
-	rm -rf super/submodule &&
-	test_must_fail git -C super submodule update >../actual &&
-	test_cmp expect actual &&
-	git -C super submodule update --checkout
-'
-
 cat << EOF >expect
 Execution of 'false $submodulesha1' failed in submodule path '../super/submodule'
 Failed to recurse into submodule path '../super'
