diff -ur git-1.7.0.3.orig/t/t1304-default-acl.sh git-1.7.0.3/t/t1304-default-acl.sh
--- git-1.7.0.3.orig/t/t1304-default-acl.sh	2010-03-22 01:35:03.000000000 +0000
+++ git-1.7.0.3/t/t1304-default-acl.sh	2010-03-23 19:53:49.069813289 +0000
@@ -9,6 +9,8 @@
 # => this must come before . ./test-lib.sh
 umask 077
 
+LOGNAME=$(whoami)
+
 . ./test-lib.sh
 
 # We need an arbitrary other user give permission to using ACLs. root

--- git-1.7.3.2/t/t3701-add-interactive.sh.orig	2010-10-22 05:49:16.000000000 +0200
+++ git-1.7.3.2/t/t3701-add-interactive.sh	2010-11-18 22:36:35.271101613 +0100
@@ -83,9 +83,9 @@
 
 test_expect_success PERL 'setup fake editor' '
 	cat >fake_editor.sh <<EOF
-	EOF
+EOF
 	chmod a+x fake_editor.sh &&
-	test_set_editor "$(pwd)/fake_editor.sh" &&
+	test_set_editor "$(pwd)/fake_editor.sh"
 '
 
 test_expect_success PERL 'dummy edit works' '
